### ISSUE TITLE
Add drag space above tabs

### ISSUE THREAD

**Issue by failsafe42** [black7375/Firefox-UI-Fix#210]
Before proton Firefox let you add extra drag space above the tabs, but now that option is gone.

Could you add back the option to have that drag space?

**Comment by black7375** [comment]
Just log (Firefox v89's css)
```css
:root {
  --space-above-tabbar: 0px;
}
:root[extradragspace][tabsintitlebar]:not([inFullscreen]) {
    --space-above-tabbar: 8px;
}
  :root[tabsintitlebar][sizemode="normal"] #toolbar-menubar[autohide="true"] {
    height: calc(var(--tab-min-height) + var(--space-above-tabbar));
  }

  /* Add extra space to titlebar for dragging */
  :root[sizemode="normal"][chromehidden~="menubar"] #TabsToolbar > .toolbar-items,
  :root[sizemode="normal"] #toolbar-menubar[autohide="true"][inactive] + #TabsToolbar > .toolbar-items {
    padding-top: var(--space-above-tabbar);
  }


#tabbrowser-tabs {
  --tabs-top-border-width: 0px;
}
#toolbar-menubar:not([autohide=true]) + #TabsToolbar,
:root:not([tabsintitlebar]),
:root[extradragspace] {
  --tabs-top-border-width: 1px;
}
:root[extradragspace] .tab-background {
  border-top-style: solid;
}
:root[tabsintitlebar]:not([extradragspace]) #toolbar-menubar[autohide=true] + #TabsToolbar .tabbrowser-tab::after,
/* Show full height tab separators on hover and multiselection. */
.tabbrowser-tab:hover::after,
#tabbrowser-tabs:not([movingtab]) > #tabbrowser-arrowscrollbox > .tabbrowser-tab[beforehovered]::after,
.tabbrowser-tab[multiselected]::after,
#tabbrowser-tabs:not([movingtab]) > #tabbrowser-arrowscrollbox > .tabbrowser-tab[before-multiselected]::after {
  margin-top: var(--tabs-top-border-width);
  margin-bottom: 0;
}
```

**Comment by black7375** [comment]
Will be supported drag space.

https://github.com/black7375/Firefox-UI-Fix/blob/49adfdc7414c077b3be8cd8b72b48aba01a7af99/userChrome.css#L667-L670
